--- vfxt.py.orig        2021-02-03 16:39:09.034784061 +0000
+++ vfxt.py     2021-02-03 17:11:23.350984152 +0000
@@ -658,7 +658,7 @@
     # generic service options
     service.POLLTIME = args.poll_time

-    if args.node_cache_size:
+    if args.node_cache_size and args.data_disk_count is None:
         if any([args.data_disk_count, args.data_disk_size]):
             logger.warning("Overriding --data-disk-count and --data-disk-size with --node-cache-size")
         disk_config = service._cache_to_disk_config(args.node_cache_size, disk_type=args.data_disk_type, machine_type=args.instance_type)